pages/_app.tsx

```diff
@@ -25,7 +35,13 @@ export default function Layout({
         <meta name="apple-mobile-web-app-status-bar" content="#536FF2" />
         <link rel="apple-touch-icon" href="/APPLE-ICON.png" />
         <link rel="icon" type="image/png" href="/APPLE-ICON.png" />
-        <link rel="manifest" href="/manifest.json" id="manifest" />
+        {username ? (
+          <link
+            rel="manifest"
+            href={`/api/${username}/manifest?${manifestParms.toString()}`}
+            id={`${username}/manifest`}
+          />
+        ) : null}
```

Comment:
I think it's better to return a default manifest in the ternary.

Reply:
I was able to resolve the issue with the icons not showing up on chrome. However if I include the default manifest the browsers will default to using it, and it is not possible to override it.